connector/MongoDBAtlasODBC.pq

```diff
@@ -36,6 +30,13 @@ MongoDBAtlasODBCType = type function (
             Documentation.SampleValues = {"db"}
         ]
     ),
+    optional loglevel as (
+        type text meta [
+            Documentation.FieldCaption = "Logging Level",
+            Documentation.FieldDescription = "Logging level for the MongoDB Atlas SQL ODBC Driver",
+            Documentation.AllowedValues = {"INFO", "ERROR", "DEBUG"}

```

Comment:
How about sorting this by increasing or decreasing verbosity?
`{"ERROR", "INFO", "DEBUG"}`

Reply:
Natacha and I spoke about this and she'd prefer them in this order. I initially had it in order of verbosity!